src/shared/ui/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+import "./SearchForm.css";
+
+export const SearchForm = ({ initialValue, onSearch, placeholder }) => {
```

Comment:
in case we don't provide initialValue let's set a default value for that 
```suggestion
export const SearchForm = ({ initialValue = "", onSearch, placeholder }) => {
```

Reply:
done